### ISSUE TITLE
0.3.5 Error opening gui

### ISSUE THREAD

**Issue by 0ptera** [LuziferSenpai/Rich-Text-Helper#9]
Opening the gui throws this error in Factorio 0.8.9:

```
Error MainLoop.cpp:1202: Exception at tick 119470310: The mod Rich Text Helper (0.3.5) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event Rich_Text_Helper::on_gui_click (ID 1)
Unknown sprite "utility/remove"
stack traceback:
	__Rich_Text_Helper__/scripts/GUI.lua:48: in function 'AddSpriteButton'
	__Rich_Text_Helper__/scripts/GUI.lua:163: in function 'RichText'
	__Rich_Text_Helper__/scripts/Main.lua:28: in function 'MainGUIToggle'
	__Rich_Text_Helper__/scripts/Main.lua:86: in function 'click'
	__Rich_Text_Helper__/scripts/Main.lua:154: in function 'handler'
	__core__/lualib/event_handler.lua:44: in function <__core__/lualib/event_handler.lua:42>
stack traceback:
	[C]: in function 'add'
	__Rich_Text_Helper__/scripts/GUI.lua:48: in function 'AddSpriteButton'
	__Rich_Text_Helper__/scripts/GUI.lua:163: in function 'RichText'
	__Rich_Text_Helper__/scripts/Main.lua:28: in function 'MainGUIToggle'
	__Rich_Text_Helper__/scripts/Main.lua:86: in function 'click'
	__Rich_Text_Helper__/scripts/Main.lua:154: in function 'handler'
	__core__/lualib/event_handler.lua:44: in function <__core__/lualib/event_handler.lua:42>
```

**Comment by LuziferSenpai** [comment]
Thanks for the report. this Sprite got removed from the base game in 18.9. So I need to see if it was intentional or not.

**Comment by LuziferSenpai** [comment]
Fixed in 0.3.6
